src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -20,176 +22,220 @@ interface Props {
   }
 }
 
-export const ArtistHeader: SFC<Props> = props => {
-  return (
-    <Subscribe to={[AppState]}>
-      {({ state }) => {
-        const {
-          mediator,
-          system: { currentUser },
-        } = state
+type Image = Props["artist"]["carousel"]["images"][0]
```

Comment:
Wasn't aware that access like this was possible 👍

Reply:
In the case of arrays it gets a little less nice, but you can still do it: `foo[0]`. I guess that in the case of the array being a tuple you can access the exact typing.

Reply:
This is also new to me, nice 👍